test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -547,6 +548,20 @@ def func(tensor):
         tensor = common_utils.get_whitenoise(sample_rate=44100)
         self._assert_consistency(func, tensor)
 
+    def test_phase_vocoder(self):
+        def func(tensor):
+            rate = 0.5
+            hop_length = 256
+            phase_advance = torch.linspace(
+                0,
+                np.pi * hop_length,
```

Comment:
Wow, I did not know that TorchScript support `np.pi` (maybe because it simply is a constant value).
However, we prefer not to depend on the third party package on the test unless that's absolutely necessary. 
Can you replace `np.pi` with something like `3.14` and remove `import numpy as np`?